services/master/controllers/challengeUpdate.go

```diff
@@ -64,25 +64,34 @@ func ChallengeUpdate(c *fiber.Ctx) error {
 	}
 
 	log.Println("Pull successful for", teamName, ". Building image...")
-
-	// Build the challenge with Dockerfile
+	FirstPatch, err := exec.Command("docker", "inspect", dir).Output()
 	cmd := exec.Command("docker", "build", "-t", dir, "./teams/"+dir)
 	cmd.Run()
-
-	// Create a labelSelector to get the challenge pod
-
-	labelSelector := metav1.LabelSelector{
-		MatchLabels: map[string]string{
-			"app": teamName + "_" + challengeName,
-		},
-	}
-	// Delete the challenge pod
-	err = client.CoreV1().Pods(namespace).DeleteCollection(context.Background(), metav1.DeleteOptions{}, metav1.ListOptions{
-		LabelSelector: metav1.FormatLabelSelector(&labelSelector),
-	})
+	cmd = exec.Command("minikube", "image", "load", dir)
+	cmd.Run()
 	if err != nil {
 		log.Println(err)
 	}
+	if len(FirstPatch) <= 3 {
```

Comment:
Why is <= 3 used?

Reply:
Because sometimes it returns an empty array and somtimes an array with 3 numbers.
not sure how inspect output is used....

Reply:
So can't we make a boolean variable `len(firstPatch) == 3` and use that in the if condition?